src/components/PlioQuestion.vue

```diff
@@ -443,47 +451,68 @@ input {
   }
 
   .btn {
-    background-color: #4caf50; /* Green */
     border: none;
     color: white;
-    padding: 2px 16px;
+    @media (orientation: portrait) {
+      padding: 2px 15px;
+    }
+    padding: 2px 40px;
     text-align: center;
     text-decoration: none;
     margin: 4px 2px;
     transition-duration: 0.4s;
     cursor: pointer;
+    height: 2.5em;
+    font-size: 1.4rem;
   }
 
   .submit {
```

Comment:
Just a note. start learning SCSS. CSS will look much cleaner!

Reply:
I thought SCSS can't run in all/old/stupid browsers. 
SCSS needs to be converted into normal CSS right? And vue does that automatically or are we using some package for that?

Reply:
Yes yes, Vue will do it automatiaclly. Or we'll need a package (that won't be included in the build -- will only run while building)

I don't think any browser can read SCSS. 

ANyway, hugo ka website ka CSS SCSS mein hi hai. Seekh hi loge :D